common/src/main/java/org/opensearch/ml/common/utils/StringUtils.java

```diff
@@ -48,13 +43,13 @@ public static String toUTF8(String rawString) {
         return utf8EncodedString;
     }
 
-    public static Map<String, Object> fromJson(String jsonStr, String defaultKey) {
+    public static Map<String, Object> fromJson(String input, String defaultKey) {
```

Comment:
any reason to change the variable name? may be we can rename to: `inputJsonStr`?

Reply:
I was going to support non json format response, so changed the parameter name for better representation. Then found we don't have such case after discussion, removed the code in function body but missed this parameter name, changed back to jsonStr.